composer.json

```diff
@@ -326,7 +326,8 @@
     "drupal/duration_field": "^1.6",
     "ckeditor/codesnippet": "^4.9",
     "drupal/file_entity": "~2.0-beta6",
-    "egulias/email-validator": "^2.1"
+    "egulias/email-validator": "^2.1",
+    "drupal/allowed_formats": "^1.3"
```

Comment:
The allowed_formats module allows us to configure a field's available text formats preventing the user from switching to a format that we don't want to support for that field. We could use this more extensively to protect fields such as impact statement which should only use very limited html.

See: https://app.zenhub.com/workspaces/journal-5ede53e8b9b11773fd13f981/issues/elifesciences/issues/6128